examples/tracing/otlp-collector/otlp-collector.ipynb

```diff
@@ -0,0 +1,135 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import threading\n",
+    "from http.server import BaseHTTPRequestHandler, HTTPServer\n",
+    "\n",
+    "from opentelemetry.proto.collector.trace.v1.trace_service_pb2 import ExportTraceServiceRequest\n",
+    "\n",
+    "\n",
+    "class OTLPCollector(BaseHTTPRequestHandler):\n",
+    "    def do_POST(self):\n",
+    "        content_length = int(self.headers['Content-Length'])\n",
+    "        post_data = self.rfile.read(content_length)\n",
+    "\n",
+    "        traces_request = ExportTraceServiceRequest()\n",
+    "        traces_request.ParseFromString(post_data)\n",
+    "        \n",
+    "        print(\"Received a POST request with data:\")\n",
+    "        print(traces_request)\n",
+    "\n",
+    "        self.send_response(200, \"Traces received\")\n",
+    "        self.end_headers()\n",
+    "        self.wfile.write(b'Data received and printed to stdout.\\n')\n",
+    "\n",
+    "def run_server(port: int):\n",
+    "    server_address = ('', port)\n",
+    "    httpd = HTTPServer(server_address, OTLPCollector)\n",
+    "    httpd.serve_forever()\n",
+    "\n",
+    "def start_server(port: int):\n",
+    "    server_thread = threading.Thread(target=run_server, args=(port,))\n",
+    "    server_thread.daemon = True\n",
+    "    server_thread.start()\n",
+    "    print(f\"Server started on port {port}. Access http://localhost:{port}/\")\n",
+    "    return server_thread"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start_server(port=4318)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from promptflow.tracing import start_trace\n",
+    "\n",
+    "os.environ[\"PF_TRACING_SKIP_LOCAL_SETUP\"] = \"true\"\n",
```

Comment:
It's not very intuitive to expose environ here. Maybe something like

start_trace(exporter=otlp_span_exporter)

Reply:
this is a workaround before we really separate `promptflow-tracing`, the real purpose here is to simulate user does not install devkit deps; but I think you are right, we might need some ways like above.